Cargo.toml

```diff
@@ -23,22 +23,25 @@ exclude = [
 all-features = true
 
 [features]
-default = ["rust_1_30", "backtraces"]
+default = ["rust_1_30", "backtraces", "std"]
+
+# Uses the `std::error::Error` trait instead of its own
+std = []
 
 # Adds the backtrace type
-backtraces = ["snafu-derive/backtraces", "backtrace"]
+backtraces = ["std", "snafu-derive/backtraces", "backtrace"]
 
 # Add conversion to `backtrace::Backtrace`
-backtrace-crate = ["backtraces"]
+backtrace-crate = ["std", "backtraces"]
 
 # Add extension traits for the futures 0.1 crate
-"futures-01" = ["futures01"]
+"futures-01" = ["std", "futures01"]
 
 # New methods on `Error`; re-export of proc-macro
 rust_1_30 = ["snafu-derive/rust_1_30", "doc-comment"]
 
 # The standard library's implementation of futures
-"unstable-futures" = ["futures-core-preview", "pin-project"]
+"unstable-futures" = ["std", "futures-core-preview", "pin-project"]
```

Comment:
`Future` is in core; why is this needed?

Reply:
I've not been looking at which other features no_std could support yet. So for now I simply marked all of them as std only. But yeah unstable-futures could probably work with no_std.

Reply:
Done